src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-mon calendar--month-length-31">
```

Comment:
Should be as in the task description
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month-length--31">
```